ps-rule.yaml

```diff
@@ -0,0 +1,56 @@
+#
```

Comment:
Do we need this in root - or should we move it into `utilities/pipelines/staticValidation`?

Reply:
Root is the default, it is anyway possible to use specific configuration files from custom locations, but that would require  to deviate from default and add options to all PSRule actions (now it's only the scheduled pipeline check, it will eventually be integrated also in module pipelines). 
Normally that is done for exceptions. So technically feasible but not sure if we should do it.

Reply:
Was just my attempt to keep root as clean as possible. It always sends a shiver down my back if I open a new project and there are like 15 top-level files which usually imply they're super important - and hence front and center :D

As per your description it sounds like 'avoid this' isn't worth the added effort we'd need - or may even complicate things more necessary. Hence I follow what ever you say makes the most sense. Just wanted to mention it :)

Reply:
Don't get me wrong, I understand the reasons behind the comment and I agree. I'd maybe suggest to open an issue to double check pros and cons of deviating from the default location once we'll get to extend module validation, would you agree? Or not even a new issue, just a comment to issue #2426

Reply:
I think commenting would be enough. It's not a biggy anyways - it just raised my eyebrow ever so slightly